api/controllers/admin.go

```diff
@@ -26,6 +30,12 @@ func (a *AdminController) Routes() []*fireball.Route {
 				"GET": a.GetConfig,
 			},
 		},
+		{
+			Path: "/admin/instancelogs",
+			Handlers: fireball.Handlers{
+				"GET": a.readInstanceLogs,
```

Comment:
Let's keep things consistent and just use `logs` instead of `instanceLogs` everywhere that applies.